Back/src/utils/users/mod.rs

```diff
@@ -0,0 +1,36 @@
+use std::{fs, path::PathBuf};
+
+use crate::database::{model::User, UserData};
+
+pub struct VerifiedUserPath {
+    _user: User,
+    _path: PathBuf,
+}
+
+impl VerifiedUserPath {
+    pub fn user(&self) -> &User {
```

Comment:
You should check/use `#[must_use]` it might be really relevant for places like theses

Reply:
This is is more of a good practice than really important but make really polish

https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute

Main takeaway: 
```
// Does not violate the unused_must_use lint.
let _ = a.user();
```
This make that it's not a big commitment as an api standpoint